docs/features/dev-card.md

```diff
@@ -16,4 +16,10 @@ You'll see the back side of your card when you click it. It shows more informati
 
 Whether you are just beginning your open source journey or a maintainer looking for more contributors and sponsors, you can share your Dev Card as a snapshot of your contributions, skills, and engagement in the open source community.
 
-Eager to get your Dev Card? Create it today and share it with your network!
+Eager to get your Dev Card? Here's how you can get yours today:
+
+1. Log in to your OpenSauced account.
+2. Navigate to your [profile](https://app.opensauced.pizza/user/{username}).
+3. Click the Dev Card icon on your profile image.
+
+![dev card icon on user profile](../../static/img/<file-name>)
```

Comment:
@Haimantika as Bekah already confirmed and you can't download the image; I've removed the SVG, downloaded the image and added it here.
One thing for you to fix, and this PR is ready! Thank you. 😊

```suggestion
![dev card icon on user profile](../../static/img/get-dev-card.png)
```